components/myguiplatform/myguirendermanager.cpp

```diff
@@ -128,8 +128,11 @@ class Drawable : public osg::Drawable {
             osg::Texture2D* texture = batch.mTexture;
             if(texture)
                 state->applyTextureAttribute(0, texture);
-
+#if defined (ANDROID) || defined (OPENGL_ES)
+            osg::GLBufferObject* bufferobject = 0;
```

Comment:
Can you comment these lines? Why can't we create `vbo->getOrCreateGLBufferObject` even if `state->isVertexBufferObjectSupported`?